qemu/tests/cfg/fio_windows.cfg

```diff
@@ -2,12 +2,13 @@
     type = fio_windows
     only Windows
     test_timeout = 600
-    install_path = "C:\fio"
     log_file = "C:\fio_log.txt"
     i386, i686:
-        install_cmd = "msiexec /a DRIVE:\fio-x86.msi /qn TARGETDIR=${install_path}"
+        install_path = "C:\Program Files (x86)\fio"
```

Comment:
it's different with the setup in Windows.cfg
so it will be install in C:\Program Files
but ${install_path} will call C:\